src/main/resources/webroot/api/openapi.yml

```diff
@@ -4,6 +4,15 @@ info:
   description: This is a Cyface data collection endpoint, that receives traffic data from Cyface measurement devices, and stores them for further processing.
   version: 0.0.0
 
+servers:
+  - url: /
+    description: Development
+  # Or else the OpenAPI requests contact /measurement on production even though it runs on /api/v*
+  - url: /api/v4
+    description: Production V4
+  - url: /api/v5
+    description: Production V5
```

Comment:
I tried to produce a log entry by sending a /measurement request from the openapi page on the staging api/v4 and noticed, that it fails with 404 because it sends the requests always to url/measurement while the api runs because of nginx e.g. on url/api/v5/measurement.

Thus, I added the option for different base urls, as we also do on the provider openapi page.